apps/api/src/app/events/usecases/run-job/run-job.usecase.ts

```diff
@@ -2,14 +2,15 @@ import { Injectable } from '@nestjs/common';
 import { JobEntity, JobRepository, JobStatusEnum } from '@novu/dal';
 import { StepTypeEnum } from '@novu/shared';
 import * as Sentry from '@sentry/node';
+
+import { RunJobCommand } from './run-job.command';
+
+import { QueueNextJob, QueueNextJobCommand } from '../queue-next-job';
+import { SendMessage, SendMessageCommand } from '../send-message';
+
 import { ApiException } from '../../../shared/exceptions/api.exception';
 import { StorageHelperService } from '../../services/storage-helper-service/storage-helper.service';
-import { QueueNextJobCommand } from '../queue-next-job/queue-next-job.command';
-import { QueueNextJob } from '../queue-next-job/queue-next-job.usecase';
-import { SendMessageCommand } from '../send-message/send-message.command';
-import { SendMessage } from '../send-message/send-message.usecase';
-import { RunJobCommand } from './run-job.command';
-import { shouldBackoff } from '../../services/workflow-queue/workflow.queue.service';
```

Comment:
Probably thanks to Nest.js dependency injection this circular dependency didn't trigger off.